Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -268,11 +263,49 @@ function IsFeatureEnabled
     return $enabledByDefault        
 }
 
+#
+# Exit if the build is a PR build and PRCA is not enabled
+#
 function ExitOnPRBuild
 {    
     if ((IsPrBuild) -and !(IsFeatureEnabled "SQPullRequestBot" $false))
     {
         Write-Host "SonarQube analysis is disabled during builds triggered by pull requests. Set a build variable named 'SQPullRequestBot' to 'true' to have the task post code analysis issues as comments in the PR. More information at http://go.microsoft.com/fwlink/?LinkID=786316"
         exit
     } 
+}
+
+#
```

Comment:
I need to reuse the logic that compares the server version so I've refactored it here. The actual logic already existed, with the exception of caching the value in a build variable.